### DISCUSSION TITLE
landing page no login button

### DISCUSSION THREAD

**Discussion by Baltimorepc** [LycheeOrg/Lychee#2545] [Bugs] [answered]
### Lychee version

v5.5.1

### Which PHP version are you using?

PHP 8.3

### Detailed description of the problem

after landing page shows i have no login button on phone or computer. Gallery shows on the right top but gets covered by tittle on phone. and still no login button showing.  As the title shows on the left side not the middle on landing page so no login button 

### Please confirm (incomplete submissions will not be addressed)

- [X] I have provided easy and step-by-step instructions to reproduce the bug.
- [X] I understand my bug report will be removed if I haven't met the criteria above.

**Comment by ildyria** [comment]
How long is your title ? :(
Can you provide a screenshot ?

And just to be clear, there are no login button on the Landing page (that is intended).

**Comment by Baltimorepc** [comment]
It is a little long when I get off I will see about getting u a screen shot I did upgrade to the alpha last night to see no login button still but the big gallery button on the middle works great for login too 
I’ll see if I can downgrade back so I can get u a screenshot before I tried the alpha release 

Sent from Yahoo Mail for iPhone


On Monday, September 23, 2024, 1:24 AM, Benoît Viguier ***@***.***> wrote:




How long is your title ? :(
Can you provide a screenshot ?

And just to be clear, there are no login button on the Landing page.

**Reply by ildyria** [reply]
While I appreciate your eagerness to be on bleeding edge, I would not recommend you do not use alpha for now. :D
I am not even running it personally. :)

**Reply by Baltimorepc** [reply]
i know and i didnt want to but i really needed to at least tempt to use it as i needed the login or gallery option in the middle so it can be seen from a phone and my fiance will ask why i haven't gotten it to work yet :) so this is me saying this is why to her so she gives me a break :)

**Reply by ildyria** [reply]
Yeah I see why this can be annoying with long titles, I haven't considered that. the Gallery link on the right would also be masked.

**Comment by Baltimorepc** [comment]
as requested here is the screen shot from edge i resized it as i am on a wide screen 
![no login on landing page](https://github.com/user-attachments/assets/45625fc0-3af8-464e-b3c3-a1177194eee5)

**Reply by Baltimorepc** [reply]
it puts the title in the left corner and no login option so i tried to switch the login to the left side but that didnt make a difference on ether landing page or main screen with landing page disabled

**Reply by ildyria** [reply, marked as answer]
That is working as intended. There is no login button on the **landing** page, only on the **gallery** page.

From the landing page, you can go to the gallery by clicking on the top right (gallery). I know super counter intuitive which is why there is now in v6 a big button "Access Gallery >>>" in the middle of the page. :)
From the Gallery then you can login.

If you do not like the landing page, you can disable it in the settings. :)

**Comment by Baltimorepc** [comment]
Yea and which is why it’s hard for users to login as they can’t see the gallery option and the login button does not show but would also be blocked since the title shows on the left on pc and middle on phone but covers the gallery option and would also cover the login if it was there 

Sent from Yahoo Mail for iPhone


On Tuesday, September 24, 2024, 1:24 AM, Benoît Viguier ***@***.***> wrote:




Yeah I see why this can be annoying with long titles, I haven't considered that. the Gallery link on the right would also be masked.

**Comment by Baltimorepc** [comment]
I understand but the landing page is needed to show so I kinda need it the big gallery button is exactly what I need as it would resolve the issue without needing to make any super changes. Could I ask when v6 may be available or just the big gallery button in the middle on 5.5.1 that would help a hole lot 


Sent from Yahoo Mail for iPhone


On Tuesday, September 24, 2024, 1:23 AM, Benoît Viguier ***@***.***> wrote:




That is working as intended. There is no login button on the landing page, only on the gallery page.

From the landing page, you can go to the gallery by clicking on the top right (gallery). I know super counter intuitive which is why there is now in v6 a big button "Access Gallery >>>" in the middle of the page. :)
From the Gallery then you can login.

If you do not like the landing page, you can disable it in the settings. :)

**Reply by ildyria** [reply]
> Could I ask when v6 may be available

My goal is end of month, but I don't know if I'll be able to meet it.

**Reply by ildyria** [reply]
@Baltimorepc  there is a Beta available and the `alpha` version on docker is quite stable now (I am using it on prod).

**Comment by Baltimorepc** [comment]
I need it for November 2nd lol but was hoping to be able to test it before then 
my fingers crossed and appreciate all the work use do for this 


Sent from Yahoo Mail for iPhone


On Tuesday, September 24, 2024, 9:55 AM, Benoît Viguier ***@***.***> wrote:





Could I ask when v6 may be available


My goal is end of month, but I don't know if I'll be able to meet it.

**Comment by Baltimorepc** [comment]
I’m using the alpha one I have seen a couple issues where it fails to upload randomly but I haven’t updated it in a couple days so maybe it’s resolved now 


Sent from Yahoo Mail for iPhone


On Wednesday, October 9, 2024, 5:11 PM, Benoît Viguier ***@***.***> wrote:




@Baltimorepc there is a Beta available and the alpha version on docker is quite stable now (I am using it on prod).

**Reply by Baltimorepc** [reply]
i just updated it Thank you

**Comment by Baltimorepc** [comment]
Alpha has been working wonderfully. Beautiful job everyone. only thing i noticed is when loading a gallery(Album) its a little slow. And when browsing a album and you select a photo to enlarge but then you wanna go back to continue browsing the album from where you left off you cant. It takes you back to the beginning of the album. Other then i haven't seen any other issues yet.

**Reply by ildyria** [reply]
yes I am aware. 
That is because we do not record the Y position of the scroll (yet) when moving and in vue those count as "page change".
I am pretty sure I know how to fix it, I just haven't had time to do it.

**Comment by Baltimorepc** [comment]
that be great :) when you have time. 

have anyone else have issues when uploading it uploads 4 of each copy.  i thought maybe it was the alpha version but i switched to the latest as you recommended but it still does it.

**Reply by ildyria** [reply]
See here: https://github.com/LycheeOrg/Lychee/discussions/2611
It is a known bug. I am trying to fix it.
